lua/cps.lua

```diff
@@ -28,6 +28,11 @@ return function (net_conf, gk_conf, gt_conf, lls_conf, numa_table)
 	-- to output routing table updates to the log.
 	cps_conf.log_level = gatekeeper.c.RTE_LOG_DEBUG
 
+	-- XXX #155 Sample parameters, need to be tested for better performance.
+	-- Log ratelimit interval and burst size.
+	cps_conf.log_ratelimit_interval_ms = 5000
```

Comment:
Drop the XXX tag. These parameters are okay and have little impact on the general performance of Gatekeeper. If someone ever wants to change them, it'll likely be in response to convenience.

This comment affects other patches in this pull request.